lisa/tools/mkfs.py

```diff
@@ -52,8 +52,8 @@ def format_disk(self, disk_name: str, file_system: FileSystem) -> None:
             raise LisaException(f"Unrecognized file system {file_system}.")
 
     def _install(self) -> bool:
-        self.node.tools[Mkfsxfs]
-        return self._check_exists()
+        # the installation is completed in format_disk based on file_system
+        return True
```

Comment:
Can it return `self._check_exists()`?

Reply:
The commit comment isn't accurate as well. It says "add missing install method". Here is not add something, it deletes logics.

Reply:
previously we don't have _install method, yesterday Diviyansh's PR was in which contains this method, I think it should return true, and the installation is completed in format_disk method.